sys/include/net/ng_netdev.h

```diff
@@ -40,6 +40,11 @@ extern "C" {
 #define NG_NETDEV_MSG_TYPE_EVENT    (0x0100)
 
 /**
+ * @brief   Flag for indicating a wireless network interface
```

Comment:
Could you also use `@name` with an appropriate description and wrap the `#define` in `@{ .. @}`?